ContextDesktop/src/test/java/com/git/migi_1/Context/entity/TestEntity.java

```diff
@@ -1,9 +1,10 @@
 package com.git.migi_1.Context.entity;
 
-import static org.junit.Assert.assertEquals;
+// import static org.junit.Assert.assertEquals;
```

Comment:
Why did you comment out this static import? For tests you usually import all these assert methods, so you don't have to type Assert.blabla.